noextern.go

```diff
@@ -72,6 +72,7 @@ func (m *ExternMetadata) Transfer(toDev, fromDev Device, v Value, synchronous bo
 	return v, nil
 }
 
+// Reset is a noop function for compatibility with the interface
```

Comment:
which interface btw?

Reply:
It's a fake "interface". Both `// +build cuda` and `// +build !cuda` requires a `Reset()` function,